examples/python/streaming.py

```diff
@@ -30,10 +30,7 @@
     .config("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog") \
     .getOrCreate()
 
-try:
-    shutil.rmtree("/tmp/delta-streaming/")
-except:
-    pass
+shutil.rmtree("/tmp/delta-streaming/", ignore_errors=Trues)
```

Comment:
Could you please change this from `ignore_error=Trues` to `ignore_error=True`

Reply:
Sure! Thanks for noticing. Done :)